source/common/http/conn_manager_impl.h

```diff
@@ -463,7 +463,11 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
       bool decoder_filters_streaming_{true};
       bool destroyed_{false};
       bool remote_complete_ : 1;
-      bool local_complete_ : 1;
+      bool local_complete_ : 1; // This indicates that local is complete prior to filter processing.
+                                // A filter can still stop the stream from being complete as seen
+                                // by the codec.
+      bool local_complete_to_codec_ : 1; // This indicates that local is complete as written all
```

Comment:
optional rename suggestion "codec_saw_local_complete_"
reads slightly better to me but the existing name is Ok if you prefer it :-)

Reply:
Sure will do.